### PR TITLE
Add complete field

### PR DESCRIPTION
**Pull request by asmacdo**
closes #3222
https://pulp.plan.io/issues/3222

### CODE REVIEW

File: pulpcore/pulpcore/app/models/repository.py [hunk 1]
```diff
@@ -239,6 +239,8 @@ class RepositoryVersion(Model):
             1 + the most recent version.
         created (models.DateTimeField): When the version was created.
         action  (models.TextField): The action that produced the version.
+        complete (models.BooleanField): False indicates that the Repository Version is still being
```
Comment (dkliban) [inline pulpcore/pulpcore/app/models/repository.py]: I think this description needs to state that the field indicates whether or not the Repository Version is user visible. Then you can say what the False means

**Comment by daviddavis** [comment]
LGTM (aside from @dkliban's suggestion)

**Comment by daviddavis** [comment]
@asmacdo I thought you were looking at a way to exclude RepositoryVersions with complete=False by default when querying them. Did that not pan out or did you decide against it?

**Comment by daviddavis** [comment]
What do you think about creating a Manager to encapsulate the complete/incomplete logic?

**Comment by asmacdo** [comment]
@daviddavis +1, complete/incomplete should be handled by pulpcore. This is an intermediate step. After demonstrating what needs to be done (this PR), I created an issue to handle this logic in core. https://pulp.plan.io/issues/3285

**Pull request merged**